tests/meta/orders/test_meta_orders_status.py

```diff
@@ -205,10 +205,10 @@ def test_store_rml(orders_api, base_store, rml_status_data, ticket: str):
 
     # WHEN storing the order
     new_pools = submitter.store_items_in_status(
-        customer=rml_status_data["customer"],
+        customer_id=rml_status_data["customer"],
```

Comment:
same as above regarding order_data.customer

Reply:
continues below

Reply:
Not in this PR